config/default.yml

```diff
@@ -581,6 +581,10 @@ Layout/ExtraSpacing:
   # things with the previous or next line, not counting empty lines or comment
   # lines.
   AllowForAlignment: true
+  # When true, allows things like 'obj.meth(arg)  # comment',
+  # rather than insisting on 'obj.meth(arg) # comment'.
+  # If done for alignment, either this OR AllowForAlignment will allow it.
+  AllowBeforeTrailingComments: false
```

Comment:
This should also be reflected in the cop description, as the manual is generated from it.

Reply:
Oh.  With all the different places to put descriptions of things it gets kinda confusing.  I'll put it there too.